### PR TITLE
Firefox: pass network tests

### PR DESCRIPTION
**Pull request by Meir017**
related to #112

### CODE REVIEW

**Comment by Meir017** [comment]
@kblok for `PlaywrightSharp.Tests.Page.Network.ResponseBufferTests` we need to fix the property `Base64Body` in the firefox protocol, it should be `base64body`

waiting for #331

File: src/PlaywrightSharp.Firefox/FirefoxRequest.cs [hunk 1]
```diff
@@ -16,7 +17,7 @@ public FirefoxRequest(FirefoxSession session, Frame frame, List<Request> redirec
             Id = payload.RequestId;
```
Comment (kblok) [inline src/PlaywrightSharp.Firefox/FirefoxRequest.cs:17]: Can we rename this guy to `FirefoxInterceptableRequest`? so we are closer to pw.

**Comment by kblok** [comment]
@Meir017 would this be in WIP? Or should we merge it and you keep working in new PRs?

**Comment by Meir017** [comment]
this is ready, I'll create more PR's for the rest

**Pull request merged**